pymarc/marc8.py

```diff
@@ -47,27 +45,42 @@ class MARC8ToUnicode:
     """
     basic_latin = 0x42
     ansel = 0x45
+
     def __init__(self, G0=basic_latin, G1=ansel, quiet=False):
         self.g0 = G0
-        self.g0_set = set([b'(', b',', b'$'])
+        self.g0_set = {b'(', b',', b'$'}
```

Comment:
I thought I knew Python well, but didn't know sets could be constructed this way. How nice!

Reply:
This is new py3 syntax that was backported to 2.7. I actually have no idea if we're trying to support py2.6 and earlier anymore. There is still code in setup.py that does things specifically on 2.5 and earlier, but we don't test on Travis on anything earlier than 2.7. Whether this is an oversight or a policy I couldn't tell you. :)

Reply:
Personally I would be happy jettisoning v <= 2.7 if it means the code can be more idiomatic and easier to support.